src/Mod/Arch/importIFCHelper.py

```diff
@@ -508,29 +501,36 @@ def getIfcPsetPoperties(ifcfile, pid):
     return getIfcProperties(ifcfile, pid, getIfcPropertySets(ifcfile, pid), {})
 
 
-# ************************************************************************************************
-def getScaling(ifcfile):
-    """returns a scaling factor from file units to mm"""
-
-    def getUnit(unit):
-        if unit.Name == "METRE":
-            if unit.Prefix == "KILO":
-                return 1000000.0
-            elif unit.Prefix == "HECTO":
-                return 100000.0
-            elif unit.Prefix == "DECA":
-                return 10000.0
-            elif not unit.Prefix:
-                return 1000.0
-            elif unit.Prefix == "DECI":
-                return 100.0
-            elif unit.Prefix == "CENTI":
-                return 10.0
-        return 1.0
+def getUnit(unit):
```

Comment:
new function? snake_case?

Reply:
This is not a new function, it's the same function that already existed. I just moved it outside another function. But yes, most functions in this module should probably be renamed; it shouldn't cause too many problems because they are just meant to be auxiliary to the importer (`importIFC`) and exporter (`exportIFC`).